### ISSUE TITLE
Localization files - Proper UTF-8 support

### ISSUE THREAD

**Issue by BujakiAttila** [JOBAA/amos-ss15-proj4#19]
In order to support our future work, we should make easier the maintenance of the localization files.

The most problematic part is the current need of escaping. Specials chars now must be inserted using an escape sequence with the hexadecimal code of the character.

It would be nice, if we could fine a solution that renders this hexadecimal escape strings unnecessary. It would be much easier if we could just enter the special chars also into the localization properties files.

**Comment by Legend-Dale** [comment]
For our implementation there is no other way than escaping.
http://forum.spring.io/forum/spring-projects/web/4512-chinese-characters-in-properties-file

This problem is specific for .properties files. Everywhere else umlauts and
s should work.

But one can use native2ascii to transform the file automatically.

I have noticed, that the Spring Tool Suite converts Chinese characters
automatically when they are typed into the file, that doesn't happen with
umlauts, though.

Maybe there is another way of doing it, such as in the Thymeleaf tutorial:
http://www.thymeleaf.org/doc/tutorials/2.1/usingthymeleaf.html

But I won't be the one implementing it. I've tried and couldn't manage to
get it work.
